File: Module.Frontend.TPM/Client/model/tpm/rejectreason/HistoricalRejectReason.js
Ext.define('App.model.tpm.rejectreason.HistoricalRejectReason', {
    extend: 'Ext.data.Model',
    idProperty: '_Id',
    breezeEntityType: 'RejectReason',
    fields: [
        { name: '_Id', type: 'string', hidden: true },
        { name: '_ObjectId', hidden: true },
        { name: '_User', type: 'string', isDefault: true },
        { name: '_Role', type: 'string', isDefault: true },
        { name: '_EditDate', type: 'date', isDefault: true },
        { name: '_Operation', type: 'string', isDefault: true },
        { name: 'Name', type: 'string', hidden: false, isDefault: true },
        { name: 'SystemName', type: 'string', hidden: false, isDefault: true }
    ],
    proxy: {
        type: 'breeze',
        resourceName: 'HistoricalRejectReasons',
        reader: {
            type: 'json',
            totalProperty: 'inlineCount',
            root: 'results'
        }
    }
});

File: Module.Persist.TPM/Model/History/HistoricalRejectReason.cs
using Core.History;
using Module.Persist.TPM.Model.TPM;
using System;

namespace Module.Persist.TPM.Model.History {
    [AssociatedWith(typeof(RejectReason))]
    public class HistoricalRejectReason : BaseHistoricalEntity<System.Guid> {
        public string Name { get; set; }
        public string SystemName { get; set; }
    }
}
